doc/whats-new.rst

```diff
@@ -37,7 +37,7 @@ Breaking changes
 
 Deprecations
 ~~~~~~~~~~~~
-
+xarray.cftime_range deprecated in favor of xarray.data_range(use_cftime=True)
```

Comment:
```suggestion
- Deprecate :py:func:`~xarray.cftime_range` in favor of :py:func:`~xarray.date_range` with ``use_cftime=True``
  (:issue:`9886`, :pull:`10024`).
  By `Josh Kihm <https://github.com/maddogghoek>`_.
```

Reply:
Feel free to word this how you please, but it would be great if you could retain the issue link, PR link, and your name.